include/util/typedefs.hpp

```diff
@@ -85,6 +85,8 @@ static const EdgeWeight INVALID_EDGE_WEIGHT = std::numeric_limits<EdgeWeight>::m
 
 using DatasourceID = std::uint8_t;
 
+using DatasourceID = std::uint8_t;
```

Comment:
Needs to be removed; is same as two lines above, or am I missing something? Why does re-aliasing the same name even compile?